src/components/FormComponents/AssetMultiPicker.js

```diff
@@ -0,0 +1,68 @@
+import React from "react";
+import PropTypes from "prop-types";
+import AssetPicker from "./AssetPicker";
+
+export default function AssetMultiPicker({
+  value,
+  onUpdate,
+  stringForm = false,
+  ...props
+}) {
+  return (
+    <>
+      {value.map((asset, index) => (
+        <div
+          key={`${asset === "native" ? "native" : "issued"}${index}`}
+          className="MultiPicker__container picker--spaceBottom"
+        >
+          <button
+            type="button"
+            className="s-button"
+            style={{ marginRight: ".5em" }}
+            onClick={() => {
+              onUpdate([...value.slice(0, index), ...value.slice(index + 1)]);
+            }}
+          >
+            ✕
+          </button>
+          <AssetPicker
+            style={{ flexGrow: "1" }}
+            value={asset}
+            stringForm={stringForm}
+            onUpdate={(newValue) => {
+              onUpdate(
+                // Immutably update the value at the current index
+                value.map((oldValue, i) => (i === index ? newValue : oldValue)),
+              );
+            }}
+            {...props}
+          />
+        </div>
+      ))}
+      <button
+        className="s-button"
+        type="button"
+        onClick={() => {
+          onUpdate(value.concat("native"));
+        }}
+      >
+        Add {value.length > 0 && "another"} asset
```

Comment:
can prob just do `Add {value.length && "another"} asset` here

Reply:
Recently I've been bit by falsy values in conditionals a few times (mostly `''`/`0` etc when I only meant it to trigger on `null` or `undefined`, but still), so I've been shooting for being explicit about what cases should trigger behavior.

Reply:
OOO I totally forgot that [nullish coalescing](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator) is officially a part of JS now